server/services/Notification.js

```diff
@@ -0,0 +1,26 @@
+class Notification {
+  _getNotificationsFromSession (session) {
+    const notifications = session.getExtraData('notification');
```

Comment:
This is more of a helper file than a class. But its ok to leave it as is.

Reply:
Haha, I don't necessarily disagree.
I came in with the assumption that it would be a lot larger than it actually ended up being.
Not that I mind having little classes, I think this might a little of a mindset difference between heavily OO java and the more flexible javascript way.